Formula/qt5.rb

```diff
@@ -86,6 +87,7 @@ class Qt5 < Formula
   depends_on :mysql => :optional
   depends_on :postgresql => :optional
   depends_on :xcode => :build
+  depends_on "openssl" => :optional
```

Comment:
👍. It's a bundle of fun. In an ideal world Qt5 would be stripped right down given the amount of hassle even the vanilla configuration causes in keeping alive and updated, let alone the options, but I've escaped having to help rescue the Qt5 formula so 😄.

Anyway, I only came to complain about Bundle doing strange things and I've ended up here, oops. Enjoy your afternoon 🙇